prime/prime_ctx.go

```
@@ -232,39 +249,55 @@ func (pc *PrimeContext) PrimeStateDBRandom(ws txcontext.WorldState, db state.Sta
 }
 
 // SelfDestructAccounts clears storage of all input accounts.
-func (pc *PrimeContext) SelfDestructAccounts(db state.StateDB, accounts []substatetypes.Address) {
+func (pc *PrimeContext) SelfDestructAccounts(accounts []substatetypes.Address) {
```

lets return the errors on lines 265, 269, 282 and 286.